publish/dir.targets

```diff
@@ -30,13 +32,13 @@
   <Target Name="GenerateDebRepoFiles"
           Condition="('$(TargetsUbuntu)' == 'true' or '$(TargetsDebian)' == 'true') and '$(TargetArchitecture)' == 'x64'">
     <ItemGroup>
-      <DebInstallerFile Include="$(PackagesOutDir)dotnet-host-*$(SharedFrameworkNugetVersion).deb" >
+      <DebInstallerFile Include="$(PackagesOutDir)dotnet-host-$(DistroRid)-$(SharedFrameworkNugetVersion).deb" >
         <PackageName>dotnet-host</PackageName>
```

Comment:
I think you should reuse these properties for this:
[https://github.com/dotnet/core-setup/blob/c54c12905b41cd5257794a2e06f7c9b5a1dc5fe4/src/pkg/packaging/dir.props#L29-L31](https://github.com/dotnet/core-setup/blob/c54c12905b41cd5257794a2e06f7c9b5a1dc5fe4/src/pkg/packaging/dir.props#L29-L31).

You may need to move them higher, or import the packaging/dir.props files.  Or refactor them out into a shared .props file.

Also, there seems to be a change here.  In the packaging props file, we use a `.` in between RID and Version.  Here you are using a `-`.